src/library_html5_webgl.js

```diff
@@ -188,15 +203,29 @@ var LibraryHtml5WebGL = {
 #if GL_DEBUG
       console.error('emscripten_webgl_create_context failed: explicitSwapControl is not supported, please rebuild with -s OFFSCREENCANVAS_SUPPORT=1 to enable targeting the experimental OffscreenCanvas specification, or rebuild with -s OFFSCREEN_FRAMEBUFFER=1 to emulate explicitSwapControl in the absence of OffscreenCanvas support!');
 #endif
+      if (resultCodePtr) {
+        HEAP32[resultCodePtr >> 2] = {{{ cDefine('EMSCRIPTEN_RESULT_NOT_SUPPORTED') }}};
+      }
       return 0;
     }
 #endif // ~!OFFSCREEN_FRAMEBUFFER
 
 #endif // ~!OFFSCREENCANVAS_SUPPORT
 
     var contextHandle = GL.createContext(canvas, contextAttributes);
+    if (resultCodePtr) {
+      if (contextHandle) {
+        HEAP32[resultCodePtr >> 2] = {{{ cDefine('EMSCRIPTEN_RESULT_SUCCESS') }}};
+      } else {
+        HEAP32[resultCodePtr >> 2] = {{{ cDefine('EMSCRIPTEN_RESULT_FAILED') }}};
+      }
+    }
     return contextHandle;
   },
+
+  emscripten_webgl_do_create_context:function(target, attributes) {
+    return LibraryHtml5WebGL.emscripten_webgl_do_create_context_ext(target, attributes, 0);
```

Comment:
```suggestion
    return _emscripten_webgl_do_create_context_ext(target, attributes, 0);
```
(`LibraryHtml5WebGL` only exists at compile time, or am I confused?)